### PR TITLE
Improved document formatting and readability

### PR DESCRIPTION
**Pull request by gitdev481**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/how-to/compare-strings.md [hunk 1]
```diff
@@ -28,9 +28,13 @@ equality, but some differences, such as case differences, may be ignored.
 
 ## Default ordinal comparisons
 
-The most common operations, <xref:System.String.CompareTo%2A?displayProperty=nameWithType> and
-<xref:System.String.Equals%2A?displayProperty=nameWithType> or <xref:System.String.op_Equality%2A?displayProperty=nameWithType> use
-an ordinal comparison, a case-sensitive comparison, and use the current
+The most common operations:
+
+**System.String.CompareTo(String)**,
```
Comment (BillWagner) [inline docs/csharp/how-to/compare-strings.md]: I really like that you have changed these to a list format.

However, I'd like to keep the link syntax. Can you replace this change with the following:

```md
- <xref:System.String.CompareTo%2A?displayProperty=nameWithType>
- <xref:System.String.Equals%2A?displayProperty=nameWithType>
- <xref:System.String.op_Equality%2A?displayProperty=nameWithType> 
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for spotting this and submitting a fix @julianstopher 

We appreciate it.  

I like the list, but want to keep the links. Can you update the PR with my suggested changes?

Thank you.

**Comment by mairaw** [comment]
I've resolved the merge conflicts but it seems @rpetrusha has changed the topic since. Do we still want to merge this @BillWagner?

**Comment by BillWagner** [comment]
Thanks for the ping @mairaw 

I'll approve and merge now. I think that additional format does make it easier to spot the important information when scanning the article.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**